include/uapi/sound/sof/tokens.h

```diff
@@ -74,6 +74,7 @@
  * #define SOF_TKN_COMP_PRELOAD_COUNT		403
  */
 #define SOF_TKN_COMP_CORE_ID			404
+#define SOF_TKN_COMP_UUID                       410
```

Comment:
why does it need to jump to 410? What's the story here?

Reply:
I originally wanted to leave some room for other uncontroversial (adding UUID to struct sof_ipc_comp was controversial at that time) tokens, but yes, let me change it to 405 now, thanks.